test/functional/test_runner.py

```diff
@@ -190,6 +190,7 @@
     'rpc_help.py',
     'feature_help.py',
     'feature_shutdown.py',
+    'feature_descriptor.py',
```

Comment:
According to the naming guide, this should be rpc_descriptor?

Reply:
Descriptor is not a RPC thing is it?

Reply:
Or `rpc_getdescriptorinfo`

Reply:
Ok, will rename to @Sjors suggestion.

Reply:
@promag Are you going to rename this or not. If not I will merge as is.

Reply:
Renamed to rpc_getdescriptorinfo.py.